### PR TITLE
Fetch all repo data before searching for cherry-picks

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: lib/tool_belt/release_environment.rb [hunk 1]
```diff
@@ -19,7 +19,8 @@ def setup(args = {})
 
       Dir.chdir(release_directory) do
         @repos.each do |name, repo|
-          @systools.execute("git clone #{repo[:repo]} #{name}") if !File.exist?(name.to_s)
+          @systools.execute("git clone #{repo[:repo]} #{name}") unless !File.exist?(name.to_s)
```
Comment (ehelms) [inline lib/tool_belt/release_environment.rb]: I feel like this changes the meaning of this statement to clone only if it already exists?

Comment (zjhuntin) [reply lib/tool_belt/release_environment.rb]: I misread the !File.exist? thought File.exist?

Comment (ehelms) [reply lib/tool_belt/release_environment.rb]: So this should be reverted?

**Comment by zjhuntin** [comment]
@ehelms updated

**Pull request merged**